jib-core/src/test/resources/cache/metadata-v3.json

```diff
@@ -0,0 +1,34 @@
+{
+  "layers": [
+    {
+      "reference": {
+        "size": 1000,
+        "digest": "sha256:5f70bf18a086007016e948b04aed3b82103a36bea41755b6cddfaf10ace3c6ef",
+        "diffId": "sha256:b56ae66c29370df48e7377c8f9baa744a3958058a766793f821dadcb144a4647"
+      }
+    },
+    {
+      "reference": {
+        "size": 2000,
+        "digest": "sha256:8c662931926fa990b41da3c9f42663a537ccd498130030f9149173a0493832ad",
+        "diffId": "sha256:a3f3e99c29370df48e7377c8f9baa744a3958058a766793f821dadcb144a8372"
+      }
+    },
+    {
+      "reference": {
+        "size": 0,
+        "digest": "sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+        "diffId": "sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+      },
+      "properties": {
+        "layerEntries": [
+          {
+            "sourceFile": "/some/source/directory",
```

Comment:
Just got this question. By just looking at the medata file, it's not possible to determine is this file should be a directory or a cache. So is it possible that Jib can be confused by this? For example, the cache saved is actually for a file, but someone was querying with the path intended to be a directory.

Reply:
> to determine is this path should be a directory or a cache

Sorry, I meant "to determine if this file should be a directory or a file."